app/bergamot-translator-app.cpp

```diff
@@ -21,19 +22,19 @@ int main(int argc, char **argv) {
   std::string config = options->asYamlString();
 
   // Route the config string to construct marian model through TranslationModel
-  TranslationModel model(config);
+  marian::bergamot::Service model(config);
 
   TranslationRequest translationRequest;
   std::vector<std::string> texts;
 
   for (std::string line; std::getline(std::cin, line);) {
-        texts.emplace_back(line);
+    texts.emplace_back(line);
   }
 
-  auto results = model.translate(std::move(texts), translationRequest);
+  auto results = model.translateMultiple(std::move(texts), translationRequest);
```

Comment:
Just saying. This change will require a change in the extension code otherwise extension code will not work out of box.

Reply:
Haven't I aliased `translate` -> `Service::translateMultiple` in bindings? I'm hoping this is the change you're talking about.

Build is failing in Extension's CircleCI (https://app.circleci.com/pipelines/github/mozilla-extensions/bergamot-browser-extension/204/workflows/ab129537-ad34-403b-b31d-40c626c19707/jobs/206/steps). But I'm guessing these are expected due to the issues discussed in https://github.com/mozilla-extensions/bergamot-browser-extension/pull/83. 

Can you confirm?

Reply:
![image](https://user-images.githubusercontent.com/727292/116564603-0b9d9100-a8fd-11eb-9520-dff10418817f.png)

Well manually downloaded XPI, seems working.

Reply:
@jerinphilip We can test it with the [wasm test page](https://github.com/browsermt/bergamot-translator/tree/main/wasm#-demo-see-everything-in-action) because that is using the latest byte based API. If it works with that then we are good to go 👍

Reply:
![image](https://user-images.githubusercontent.com/727292/116736462-d4f07500-a9e7-11eb-8f1d-a9ed982b9b27.png)

I have no clue what is happening, but I am assuming bindings transfer is intact, given I/O works?

Reply:
> @jerinphilip We can test it with the [wasm test page](https://github.com/browsermt/bergamot-translator/tree/main/wasm#-demo-see-everything-in-action) because that is using the latest byte based API. If it works with that then we are good to go 👍

@jerinphilip Let me help you with testing. I will test it at my end and confirm 👍

Reply:
@jerinphilip I tested your change with the fix in https://github.com/browsermt/bergamot-translator/pull/125 and it works 👍

Reply:
I have pulled #125 into a [test build](https://github.com/browsermt/bergamot-translator/tree/jp/collapse-yolo), still translates to "schwanger". I guess the transfer is intact from your response and this is something to resolve at my end maybe.